modules/logging-collector-alerts.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/default-logging-alerts.adoc
+
+:_content-type: REFERENCE
+[id="logging-collector-alerts_{context}"]
+= Logging collector alerts
+
+In logging 5.8 and later versions, the following alerts are generated by the {clo}. You can view these alerts in the {product-title} web console.
+
+[cols="4", options="header"]
+|===
+| Alert Name | Message | Description | Severity
+
+| CollectorNodeDown
+| Prometheus could not scrape `namespace`/`pod` collector component for more than 10m.
+| Collector cannot be scraped.
+| Critical
+
+| CollectorHighErrorRate
+| `value`% of records have resulted in an error by `namespace`/`pod` collector component.
```

Comment:
```suggestion
| The `<value>`% of records have resulted in an error by `namespace`/`pod` collector component.
```

Reply:
`namespace`/`pod  > the slash is confusing. I'm unsure if it's part of the naming convention, used as an or statement, or stating "per"?